From 237c3a2c4273262a318c8b64cecf5f8c1274a279 Mon Sep 17 00:00:00 2001
From: Tomas Henzl <thenzl@redhat.com>
Date: Mon, 8 Sep 2014 14:31:41 -0400
Subject: [scsi] hpsa: update driver version to 3.4.4-1

Message-id: <1410186734-19986-51-git-send-email-thenzl@redhat.com>
Patchwork-id: 92070
O-Subject: [RHEL6.6 e-stor PATCH 50/83] hpsa: update driver version to 3.4.4-1
Bugzilla: 1069185
RH-Acked-by: Maurizio Lombardi <mlombard@redhat.com>
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Ewan Milne <emilne@redhat.com>

Signed-off-by: Stephen M. Cameron <scameron@beardog.cce.hp.com>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
(cherry picked from commit 9a993302cc7a2e0a22e0851122dcfb59b56abd7a)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

Conflicts:
 drivers/scsi/hpsa.c
 (version string is RH specific)

diff --git a/drivers/scsi/hpsa.c b/drivers/scsi/hpsa.c
index e194de6..5fe0483 100644
--- a/drivers/scsi/hpsa.c
+++ b/drivers/scsi/hpsa.c
@@ -53,7 +53,7 @@
 #include "hpsa.h"
 
 /* HPSA_DRIVER_VERSION must be 3 byte values (0-255) separated by '.' */
-#define HPSA_DRIVER_VERSION "3.4.2-RH5"
+#define HPSA_DRIVER_VERSION "3.4.4-1-RH1"
 #define DRIVER_NAME "HP HPSA Driver (v " HPSA_DRIVER_VERSION ")"
 #define HPSA "hpsa"
 
-- 
1.7.1